### ISSUE TITLE
Obsolete methods

### ISSUE THREAD

**Issue by rdewijn** [tercen/log_cutoff_operator#2]
This operator contains a lot of obsolete methods, makes it overly complicated.

- Quantile shift method is obsolete
- Dependency on pgCheckInput is obsolete, I guess.

Propose to make a simple operator that performs log transformation, with the default option to cut-off negative values at a pre-specified level (cut method), and fails with multiple values per cell.

The issue with multiple values per cell and the option to bypass log transformation play a role in the LFC app. probabaly better to handle these there separately?

**Comment by ginberg** [comment]
@fnaji what's your take on this?

**Comment by ginberg** [comment]
hi @rdewijn, the operator has been simplified: Quantile shift is removed, pgCheckInput dependency is removed and it fails with multiple values per cell. Can you have a look at it again?

**Comment by rdewijn** [comment]
@ginberg the shift_offset was a component of the quantile shift method. Remove as well. Cut-off and log-base remain as properties.

**Comment by ginberg** [comment]
@rdewijn ok, I understand about the shift_offset. Should I also remove the 'Treatment of negative values' property?

**Comment by rdewijn** [comment]
@ginberg  Indeed, 'Treatment_ of negative values' property is not strictly necessary (`none` could  be replaced by setting cut-off = 0), but may be good to keep anyway to make explicit that negative values are (by default) replaced.
